pages/index.tsx

```diff
@@ -1,15 +1,13 @@
-import { signIn, useSession } from "next-auth/react"
+import { useSession } from "next-auth/react"
 
 function HomePage() {
   const { data: session } = useSession()
 
-  return <div>
-    Welcome to SalesWell{session ? ' ' + session.user.name : ''}!
-    <br />
-    { !session ? 
-      <button onClick={() => signIn()}>Sign in with Email</button> 
-      : null }
-  </div>
+  return (
+    <div className="body">
+      <h1 className="title">Welcome{session ? ' back, ' + session.user.name + ' !' : ' to SalesWell'}</h1>
```

Comment:
Try not to write more than 80 characters on a single line, if you do, continue on the next line with indentation.
Also, put a space after `{` and before `}`